BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.m

```diff
@@ -22,6 +22,11 @@
 #import "BOXUserRequest.h"
 #import "BOXContentClient+User.h"
 
+// Default API URLs
+NSString *const BOXDefaultAPIBaseURL = @"https://api.box.com/2.0";
```

Comment:
Sounds good. Maybe still a comment in the accessors that the 'APIBaseURL' and 'APIUploadBaseURL' are expected to contain the API version number within them.